.github/labeler.yml

```diff
@@ -0,0 +1,99 @@
+clients:
+- changed-files:
+  - any-glob-to-any-file: 
+    - 'clients/**'
+
+config:
+- changed-files:
+  - any-glob-to-any-file:
+    - 'config/**'
+
+connect:
+- changed-files:
+  - any-glob-to-any-file: 
+    - 'connect/**'
+
+core:
+- changed-files:
+  - any-glob-to-any-file:
+    - 'core/**'
+
+raft:
+- changed-files:
+  - any-glob-to-any-file: 
+    - 'raft/**'
+
+storage:
+- changed-files:
+  - any-glob-to-any-file: 
+    - 'storage/**'
+
+streams:
+- changed-files:
+  - any-glob-to-any-file: 
+    - 'streams/**'
+
+tests:
+- changed-files:
+  - any-glob-to-any-file: 
+    - 'tests/**'
```

Comment:
remove this one

Reply:
Keep this one to reminder me temporarily until I find a new method to add tests label like @chia7712 said.

Reply:
Ok. "minor" and "tests" will complete next part.

Reply:
Hi @mumrah , I think https://github.com/apache/kafka/pull/17208/commits/91bafe1d6015c058056978d4d5ff4acd264e04f9 meet @chia7712 requirement.
PTAL.